packages/tasit-sdk/src/testHelpers/mochaSetup.js

```diff
@@ -0,0 +1,46 @@
+// Chai
+import { expect } from "chai";
+global.expect = expect;
+
+// Helpers
+import helpers from "./helpers";
+global = Object.assign(global, helpers);
+
+// External helpers
+//
+// Note:  Using dist file because babel doesn't compile node_modules files.
+// Any changes on src should be followed by compilation to avoid unexpected behaviors.
+// Note that lerna bootstrap does this for you since it
+// runs prepare in all bootstrapped packages.
+// Refs: https://github.com/lerna/lerna/tree/master/commands/bootstrap
+import { createFromPrivateKey } from "tasit-account/dist/testHelpers/helpers";
+global.createFromPrivateKey = createFromPrivateKey;
+
+import { Action } from "../TasitSdk";
+const { ConfigLoader } = Action;
+import config from "../config/default.js";
+ConfigLoader.setConfig(config);
+
+// Global hooks
+let snapshotId;
+
+beforeEach("global beforeEach() hook", async () => {
```

Comment:
Is this running before tests that don't even involve `ganache-cli` like the `tasit-account` tests?

Reply:
Extracted to: https://github.com/tasitlabs/TasitSDK/issues/237